src/UnoDrive.Shared/ViewModels/MyFilesViewModel.cs

```diff
@@ -169,7 +176,6 @@ async Task LoadDataAsync(string pathId = null, Action callback = null)
 			}
 			finally
 			{
-				cancellationTokenSource.Dispose();
```

Comment:
I believe that you don't need to call `Dispose` here, makes sense to call the `Dispose` when we set a timeout, as you can [see here](https://github.com/davidfowl/AspNetCoreDiagnosticScenarios/blob/master/AsyncGuidance.md#always-dispose-cancellationtokensources-used-for-timeouts).
 
But I presume that's ok to call the `Dispose` anyway